frontend/lib/src/components/elements/LinkButton/BaseLinkButton.tsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { ReactElement } from "react"
+import {
+  BaseLinkButtonProps as BaseLinkButtonPropsT,
+  BaseButtonKind,
+  BaseButtonSize,
+  StyledPrimaryLinkButton,
+  StyledSecondaryLinkButton,
+} from "./styled-components"
+
+function BaseLinkButton({
```

Comment:
nit: The reason why this isn't integrated into shared BaseButton is probably because it requires an `a` tag instead of a `button` or? Maybe add the reason as a comment as well

Reply:
Done